scala.wake

```diff
@@ -15,22 +15,53 @@ def scalaPluginPath = "wit/lib"
 global def readIvyDepsJSON dir =
   source "{dir}/ivydependencies.json" | parseJSONFile
 
-# Should be installed by wit
```

Comment:
There's also an additional 'installed by wit' earlier in the file (for whatever reason GH wont let me comment there)

Reply:
Thanks for noticing this, will fix